src/main/java/com/graphaware/module/es/search/resolver/PropertyResolver.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2013-2016 GraphAware
+ *
+ * This file is part of the GraphAware Framework.
+ *
+ * GraphAware Framework is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of
+ * the GNU General Public License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+package com.graphaware.module.es.search.resolver;
+
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Result;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
```

Comment:
Right now this resolver is not usable.
It need to be tested and added to `ResolverFactory.createResolver` to be usable.
I kept this class for future implementation and testing only.